airbyte-integrations/connectors/source-amazon-ads/source_amazon_ads/manifest.yaml

```diff
@@ -1,168 +1,1195 @@
+            recommendationType: BIDS_FOR_EXISTING_AD_GROUP
+        partition_router:
+          type: SubstreamPartitionRouter
+          parent_stream_configs:
+            - type: ParentStreamConfig
+              parent_key: "adGroupId"
+              stream: "#/definitions/streams/sponsored_product_ad_groups"
+              partition_field: "adGroupId"
+              extra_fields:
+                - ["campaignId"]
+        record_selector:
+          type: RecordSelector
+          extractor:
+            type: DpathExtractor
+            field_path: ["bidRecommendations"]
+        paginator:
+          type: DefaultPaginator
+          page_token_option:
+            type: RequestOption
+            inject_into: body_json
+            field_name: nextToken
+          page_size_option:
+            type: RequestOption
+            inject_into: body_json
+            field_name: maxResults
+          pagination_strategy:
+            type: CursorPagination
+            page_size: 100
+            cursor_value: "{{ response.nextToken }}"
+      transformations:
+        - type: AddFields
+          fields:
+            - path: ["adGroupId"]
+              value: '{{ stream_slice["adGroupId"] }}'
+            - path: ["campaignId"]
+              value: '{{ stream_slice.extra_fields["campaignId"] }}'
+
+    sponsored_product_ad_group_suggested_keywords:
+      type: DeclarativeStream
+      name: sponsored_product_ad_group_suggested_keywords
+      primary_key: []
+      retriever:
+        type: SimpleRetriever
+        requester:
+          $ref: "#/definitions/base_requester"
+          error_handler: "#/definitions/error_handler"
+          path: "v2/sp/adGroups/{{ stream_slice['adGroupId'] }}/suggested/keywords"
+          http_method: GET
+          request_headers:
+            Amazon-Advertising-API-Scope: "{{ stream_slice.parent_slice['profileId'] }}"
+            Amazon-Advertising-API-ClientId: '{{ config["client_id"] }}'
+          request_params:
+            maxNumSuggestions: 100
+        partition_router:
+          type: SubstreamPartitionRouter
+          parent_stream_configs:
+            - type: ParentStreamConfig
+              parent_key: "adGroupId"
+              stream: "#/definitions/streams/sponsored_product_ad_groups"
+              partition_field: "adGroupId"
+              extra_fields:
+                - ["campaignId"]
+        record_selector:
+          type: RecordSelector
+          extractor:
+            type: DpathExtractor
+            field_path: []
+        paginator:
+          type: NoPagination
+
+    attribution_report_products:
+      type: DeclarativeStream
+      name: attribution_report_products
+      primary_key: []
+      retriever:
+        type: SimpleRetriever
+        requester:
+          $ref: "#/definitions/base_requester"
+          error_handler:
+            type: DefaultErrorHandler
+            response_filters:
+              - http_codes: [400]
+                action: IGNORE
```

Comment:
We seem to only ignore this one if the message is `This profileID is not authorized to use Amazon Attribution`. Should we add this condition here too?

Reply:
updated